packages/treasury/src/vault.js

```diff
@@ -58,6 +58,8 @@ export function makeVaultKit(
     assert(vaultState === VaultState.ACTIVE, 'vault must still be active');
   }
 
+  console.log(`VALT   ${q(manager)}`);
```

Comment:
This looks like debug logging. Any particular reason to leave it turned on in production?